src/app/application/services/i18n/I18n.ts

```diff
@@ -61,7 +61,7 @@ export default <T extends I18nRecord>(strs: T, env: Env['var']) => {
             // temporarily change any http.api terms to camelCase at runtime
             return camelCaseToWords((e as I18nError).key.split('.').pop()!)
           default:
-            if (rest[2]?.defaultMessage) {
+            if (typeof rest[2]?.defaultMessage === 'string') {
```

Comment:
Previously this being Javascript `''` would evaluate to `false`, meaning we couldn't use `''` as a default message when a i18n string wasn't found. We now specifically check for a string instead of truthyness.

You'll see where I use a `defaultMessage: ''` in EmptyBlock further down in this PR.